assets/js/components/help/HelpMenuLink.js

```diff
@@ -21,12 +21,24 @@
  */
 import PropTypes from 'prop-types';
 
+/**
+ * WordPress dependencies
+ */
+import { useCallback } from '@wordpress/element';
+
 /**
  * Internal dependencies
  */
 import Link from '../Link';
+import { trackEvent } from '../../util';
+
+function HelpMenuLink( { children, href, gaEventLabel } ) {
+	const onClick = useCallback( async () => {
+		if ( gaEventLabel ) {
+			await trackEvent( 'global_help_menu', 'click_outgoing_link', gaEventLabel );
+		}
+	}, [] );
```

Comment:
@Hazlitte we need to use `gaEventLabel` as a dependency for this callback.

Reply:
👍